core/src/ldml/ldml_transforms.cpp

```diff
@@ -398,26 +411,90 @@ transform_entry::transform_entry(const std::u32string &from, const std::u32strin
 
 size_t
 transform_entry::match(const std::u32string &input) const {
-  if (input.length() < fFrom.length()) {
-    // TODO-LDML: regex
-    // Too small, can't match.
-    return 0;
+  // TODO-LDML: simple approach, new regex every time
```

Comment:
I presume by this you mean you'll later precompile all regexes at keyboard load time?
```suggestion
  // TODO-LDML: simple approach, new regex every time -- need to precompile regexes at keyboard load
```

Reply:
yes. In fact,  already precompiled. The Matcher object could be cached, but that will be more of a memory rather than time tradeoff